mmv1/templates/terraform/examples/firebase_extentions_instance_resize_image.tf.erb

```diff
@@ -0,0 +1,63 @@
+resource "google_storage_bucket" "images" {
+  provider                    = google-beta
+  project                     = "<%= ctx[:test_env_vars]['project_id'] %>"
+  name                        = "<%= ctx[:vars]['bucket_id'] %>"
+  location                    = "US"
+  uniform_bucket_level_access = true
+
+  # Delete all objects when the bucket is deleted
+  force_destroy = true
+}
+
+resource "google_service_account" "resize_image" {
```

Comment:
@pavelgj @johan I'm getting the following permission denied error on CreateExtensionInstance. Is there another permission missing somewhere? Does the Extensions API use a service account and needs to be able to act as this service account just created?

```
generic::permission_denied: Service account `tf-test-sa@<project-id>.iam.gserviceaccount.com` does not exist in `projects/<project-id>` or the caller does not have the correct permissions to use this service account [google.rpc.error_details_ext] { code: 7 message: \"Service account `tf-test-sa@<project-id>.iam.gserviceaccount.com` does not exist in `projects/<project-id>` or the caller does not have the correct permissions to use this service account\"
```